src/lumigo_tracer/lumigo_utils.py

```diff
@@ -487,7 +487,7 @@ def aws_dump(d: Any, decimal_safe=False, **kwargs) -> str:
 
 
 def lumigo_dumps(
-    d: Union[bytes, str, dict, OrderedDict, list],
+    d: Optional[Union[bytes, str, dict, OrderedDict, list]],
```

Comment:
IMO, but however you prefer :)
```suggestion
    d: Union[bytes, str, dict, OrderedDict, list, None],
```